src/index.js

```diff
@@ -222,6 +222,26 @@ export class ULabel {
         return ret;
     }
 
+    //returns a list of confidence values and ids of a subtask
+    static get_annotation_confidence(subtask) {
```

Comment:
Switch this to operate on individual annotations and put it in it's own file, maybe something like `annotation_operators.ts`.